frontend/src/icon/ArrowRight.js

```diff
@@ -0,0 +1,19 @@
+import { FiChevronRight } from "react-icons/fi";
+import styled from "styled-components/macro";
+
+export default function ArrowRight() {
+    return (
+
+        <Arrow>
+            <FiChevronRight/>
+        </Arrow>
```

Comment:
do you need the surrounding `<Arrow>` tags if they already represent a `FiChevronRight` tag?